components/tools/OmeroWeb/omeroweb/webclient/templates/webclient/annotations/metadata_general.html

```diff
@@ -746,7 +746,7 @@ <h1 class="can-collapse closed" data-name="tags">
             <!-- MAP ANNOTATIONS -->
             <div id="mapAnnsPane">
             <h1 class="can-collapse closed" data-name="keyvaluepairs">
-                Key-Value Pairs
+                Attributes
```

Comment:
No worries. But the change was requested. So we either leave a difference between the m52 branch and develop or we consider it suitable for configuration. My preference would be the latter.